frontend/apps/docs/content/docs/cli/index.mdx

```diff
@@ -6,6 +6,8 @@ import { Tab, Tabs } from 'fumadocs-ui/components/tabs'; // For package-install
 
 ## Quick Start
 
+For sample projects and setup examples, check out our [liam-erd-samples](https://github.com/liam-hq/liam-erd-samples) repository.
+
```

Comment:
I thought it would be clearer to make the description of the sample repository as a section at the very end, what do you think?